src/Client.js

```diff
@@ -281,6 +282,22 @@ Client.prototype.close = function(opts) {
   return this._http.close(opts)
 }
 
+/**
+ * Executes a query via the FaunaDB Query API.  
+ * See the [docs](https://app.fauna.com/documentation/reference/queryapi),
+ * and the query functions in this documentation.
+ * @param expression {module:query~ExprArg}
+ *   The query to execute. Created from {@link module:query} functions.
+ * @param {?Object} options
+ *   Object that configures the current query, overriding FaunaDB client options.
+ * @param {?string} options.secret FaunaDB secret (see [Reference Documentation](https://app.fauna.com/documentation/intro/security))
+ * @return {external:Promise<Object>} An object containing the FaunaDB response object and the list of query metrics incurred by the request.
+ */
+ Client.prototype.queryWithMetrics = function(expression, options) {
+  options = Object.assign({}, options, { withMetrics: true })
```

Comment:
this is using `withMetrics` but elsewhere `metrics` is used.

```suggestion
  options = Object.assign({}, options, { metrics: true })
```